### PR TITLE
Moved kafka metrics into kafka handler

### PR DESCRIPTION
**Pull request by Gobonoid**

### CODE REVIEW

File: proximo-server/server.go [hunk 1]
```diff
@@ -114,7 +113,7 @@ func (s *server) Consume(stream MessageSource_ConsumeServer) error {
 
 	select {
 	case err := <-errors:
-		s.counters.ErrorCounter.Inc()
+		s.counters.	ErrorCounter.Inc()
```
Comment (ggirtsou) [inline proximo-server/server.go]: no need for whitespace here

**Comment by ggirtsou** [comment]
As I don't have context for this change, can you mention what problem you're trying to solve with this change, please?

**Comment by Gobonoid** [comment]
@ggirtsou Currently metrics doesn't behave correctly. This is meant to fix that behaviour.

(Metrics counts only once, per rebalancing not per message consumption)

**Review by jakekeeys — COMMENTED** [review]
what happened to the sink counter

**Comment by Gobonoid** [comment]
@jakekeeys thanks Jake. That was mistake cp, got it cleared now.

**Review by jakekeeys — APPROVED** [review]
LGTM.... @mjgarton?

**Review by mjgarton — APPROVED** [review]

**Pull request merged**